Hunting Queries/BehaviorAnalytics/Anomalous Role Assignment.yaml

```diff
@@ -26,5 +26,5 @@ query: |
  | where isnotempty(RoleId) or isnotempty(RoleName)
  | join kind=inner ( BehaviorAnalytics
  ) on $left._ItemId == $right.SourceRecordId
- | where UsersInsights.BlasrRadius == "High" or ActivityInsights.FirstTimeUserPerformedAction == true
+ | where UsersInsights.BlasrRadius == "High" or ActivityInsights contains "True"
```

Comment:
It is possible to use "has" here instead of "contains".  It is much more efficient.

Reply:
done